atomic_reactor/plugins/pre_resolve_composes.py

```diff
@@ -131,6 +131,12 @@ def adjust_for_autorebuild(self):
             self.log.info('Autorebuild detected: Ignoring compose_ids plugin parameter')
             self.compose_ids = tuple()
 
+    def get_arches(self):
+        # This will be replaced by a plugin soon but this will do for now
+        for plugin in self.workflow.buildstep_plugins_conf:
+            if plugin['name'] == PLUGIN_BUILD_ORCHESTRATE_KEY:
+                return plugin.get('args').get('platforms', [])
```

Comment:
If 'args' isn't present, `.get('args')` will return `None` anyway leading to a traceback when we try to call `.get` on it, so we may as well use `plugin['args']`.
If the plugin isn't found, we'll return `None`, but if the plugin is found but doesn't have a 'platforms' argument we'll return `[]`. Probably better to fail in that second (invalid config) case.
So I'd suggest: `return plugin['args']['platforms']`

Reply:
Fixed.